File: ParkingFlow.Server/Dto/DashboardData.cs
namespace ParkingFlow.Server.Dto;

public class DashboardData
{
    public int TotalParkingAreas { get; set; }
    public int TotalParkingFees { get; set; }
    public int TotalParkingAreasActive { get; set; }
    public decimal LastMonthEarningsTotalUsd { get; set; }
    public decimal CurrentMonthEarningsTotalUsd { get; set; }
    public List<ParkingHistoryPayment>? ParkingHistoryPayments { get; set; } = null;

}

File: parkingflow.client/src/models/DashboardData.ts
import { ParkingHistoryPayment } from "@/wrappers/ParkingHistoryPayment";

export interface DashboardData {
    totalParkingAreas: number;
    totalParkingFees: number;
    totalParkingAreasActive: number;
    lastMonthEarningsTotalUsd: number;
    currentMonthEarningsTotalUsd: number;
    parkingHistoryPayments?: ParkingHistoryPayment[];
}